plugins/default/about.js

```diff
@@ -6,7 +6,7 @@ var about = {
     permission: ['isListener']
  },
  onCall: function(request) {
-    request.sendChat('This broadcast is currently running "Grooveshark Broadcast Bot" ' + GLOBAL.GSBOTVERSION + ', created by grooveshark.com/uman42 ~ github.com/UlysseM/GSBOT/');
+    request.sendChat('This broadcast is currently running "Grooveshark Broadcast Bot" ' + GLOBAL.GSBOTVERSION + ', created by grooveshark.com/uman42 and grooveshark.com/pironic, which is open source! Got a feature or code fix? Submit it to our open repository via pull request at github.com/UlysseM/GSBOT/');
```

Comment:
Shit, I never planned on including that change in any pull requests. Sorry
about that. I'm glad you caught it, and that you can address my intention
any way.

That would be really cool of you. I know that I've done like 500 lines of
code to your 7000 lines and wouldn't expect my name on the core, but
showing the names of plugin authors would be cool too. :-)

I'm a little embarrassed, and humbled by your suggestion. Thanks man.

On Tue, 24 Mar 2015 5:20 pm Ulysse Manceron <notifications@github.com>
wrote:

> In plugins/default/about.js
> <https://github.com/UlysseM/GSBOT/pull/40#discussion_r27083210>:
>
> > @@ -6,7 +6,7 @@ var about = {
> >      permission: ['isListener']
> >   },
> >   onCall: function(request) {
> > -    request.sendChat('This broadcast is currently running "Grooveshark Broadcast Bot" ' + GLOBAL.GSBOTVERSION + ', created by grooveshark.com/uman42 ~ github.com/UlysseM/GSBOT/');
> > +    request.sendChat('This broadcast is currently running "Grooveshark Broadcast Bot" ' + GLOBAL.GSBOTVERSION + ', created by grooveshark.com/uman42 and grooveshark.com/pironic, which is open source! Got a feature or code fix? Submit it to our open repository via pull request at github.com/UlysseM/GSBOT/');
>
> My latest commit should address that issue.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UlysseM/GSBOT/pull/40/files#r27083210>.
>